data/alexandria start.txt

```diff
@@ -0,0 +1,1290 @@
+				goto questions
+			label accept
+			`	"Finally."`
+				accept
+	on visit
+		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
+
+
+
+
+mission "Alexandria Library Card: Rescue"
+	name "Alexandria Library Card"
+	waypoint Mora
+	description `Repair the "Gonza Book Cruise" in the Mora system and take it to <destination> before <date>.`
+	blocked `Daemos informs you, "I don't fit in a glove compartment. I need a bunk and space for 1 ton of cargo."`
+	source Valhalla
+	destination Ingot
+	passengers 1
+	landing
+	deadline 11
+	cargo "old books and a thousand pencils" 1
+	on offer
+		conversation
+			`A Deep Security officer runs towards you and Daemos in a hurry yelling, "It's been disabled! We have to leave!"`
+			`	When the officer reaches you, he explains the Alexandria Seeker who was supposed to meet you is in grave trouble. The Seeker's ship is disabled in an uninhabited system called Mora.' The Seeker, Gonza Doorin, refuses to let anyone onto his ship except another Seeker. There is a pirate fleet in the system, attacking anyone who gets near the ship."`
+			`	Daemos interjects, "Can Deep Security escort us?"`
+			`	"Of course," the officer responds. "We must leave at once.`
+			`	"Hurry up and finish your business here. Meet my fleet in the spaceport."`
+			choice
+				`	"Agreed."`
+					accept
+				`	"Rescue your own ship. I have better things to do than save self-important librarians."`
+					decline
+	to offer
+		has "Alexandria Library Card: Return: done"
+	on complete
+		payment 60000
+	on visit
+		dialog `You need to bring Daemos, his cargo, and the Gonza Book Cruise here to complete this mission.`
+	on decline
+		fail "Alexandria Library Card: Rescue: Escort"
+	on fail
+		fail "Alexandria Library Card: Rescue: Escort"
+	npc accompany save
+		personality escort derelict timid target
+		government "Librarian"
+		ship "Star Queen" "Gonza Book Cruise"
+		system Mora
+	npc
+		government "Pirate"
+		personality staying plunders disables
+		system Mora
+		fleet "Small Southern Pirates" 3
+		fleet "Large Northern Pirates" 2
+
+
+
+
+mission "Alexandria Library Card: Rescue: Escort"
+	name `Escort Fleet: Deep Security`
+	description `A Deep Security rescue fleet will escort you to Mora so you can repair the Gonza Book Cruise before <date>.`
+	source Valhalla
+	destination Ingot
+	deadline 9
+	to offer
+		has "Alexandria Library Card: Return: done"
+		not "Alexandria Library Card: Rescue: declined"
+		not "Alexandria Library Card: Rescue: failed"
+	on offer
+		conversation
+			`Deep Security meets you and Daemos. "A fleet is prepared, but only your ship can board the Gonza Book Cruise. We must bring it back to <destination> before <date>."`
```

Comment:
```suggestion
			`Deep Security meets you and Daemos. "A fleet is prepared, but only your ship can board the Gonza Book Cruise. We must be back before <date>."`
```

Reply:
The ship's destination should be specified. Also, the destination is not where you are at the time that statement is made. However, "back to" is incorrect as well.

Reply:
Fixed.